apps/back/templates/expense/list.html.twig

```diff
@@ -5,5 +5,31 @@
 {% endblock %}
 
 {% block body %}
-    <h1>Liste des dépenses</h1>
+    <h2>Groupe : {{ group.getName() }}</h2>
+    <h2>Liste des dépenses</h2>
+    <div>
+        <table>
+            <tr>
+                <th>Nom<br>Payé par</th>
```

Comment:
The usage of the `<br>` tag is not recommended to visually add breaking line in your document. It has an impact on accessibility browsers and can easily replaced by a css rule.